plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveColumnProjectionInfo.java

```diff
@@ -109,9 +108,7 @@ public boolean equals(Object obj)
 
     public static String generatePartialName(List<String> dereferenceNames)
     {
-        return dereferenceNames.stream()
-                .map(name -> "#" + name)
-                .collect(Collectors.joining());
+        return String.join("#", dereferenceNames);
```

Comment:
This change returns a different result from the existing method. The previous method had a leading `#`. Are you changing it intentionally?

Reply:
Good catch. No, the change was not done intentionally.

Reply:
CI failure relates to this change. Could you remove the commit from PR?